src/hooks/useSystemIntakeContacts.tsx

```diff
@@ -1,110 +1,165 @@
-        commonName: systemIntake.businessOwner.name || '',
-        component: systemIntake.businessOwner.component || ''
+        commonName: businessOwner?.name || '',
+        component: businessOwner?.component || '',
+        systemIntakeId
       },
       productManager: {
         ...initialContactsObject.productManager,
-        commonName: systemIntake.productManager.name || '',
-        component: systemIntake.productManager.component || ''
+        commonName: productManager?.name || '',
+        component: productManager?.component || '',
+        systemIntakeId
       },
       isso: {
         ...initialContactsObject.isso,
-        commonName: systemIntake.isso.name || ''
+        commonName: isso?.name || '',
+        systemIntakeId
       }
     };
 
     // Return formatted contacts
     return systemIntakeContacts.reduce<FormattedContacts>(
       (contactsObject, contact) => {
+        // If contact is primary role, add to object
         if (rolesMap[contact.role as Role]) {
           return {
             ...contactsObject,
             [rolesMap[contact.role as Role]]: contact
           };
         }
+        // If contact is additional contacts, add to additional contacts array
         return {
           ...contactsObject,
-          additionalContacts: [...contactsObject.additionalContacts, contact]
+          additionalContacts: [
+            ...contactsObject.additionalContacts,
+            {
+              ...contact,
+              commonName: contact.commonName || '',
+              email: contact.email || '',
+              systemIntakeId
+            }
+          ]
         };
       },
       mergedContactsObject
     );
-  }, [data?.systemIntakeContacts?.systemIntakeContacts, systemIntake]);
+  }, [systemIntakeContacts, systemIntake, systemIntakeId]);
 
   const [
     createSystemIntakeContact
-  ] = useMutation<CreateSystemIntakeContactInput>(CreateSystemIntakeContact);
+  ] = useMutation<CreateSystemIntakeContactPayload>(CreateSystemIntakeContact);
   const [
     updateSystemIntakeContact
-  ] = useMutation<UpdateSystemIntakeContactInput>(UpdateSystemIntakeContact);
+  ] = useMutation<UpdateSystemIntakeContactPayload>(UpdateSystemIntakeContact);
   const [
     deleteSystemIntakeContact
-  ] = useMutation<DeleteSystemIntakeContactInput>(DeleteSystemIntakeContact);
+  ] = useMutation<DeleteSystemIntakeContactPayload>(DeleteSystemIntakeContact);
 
-  /** Create system intake contact in database */
-  const createContact = async (contact: SystemIntakeContactProps) => {
+  /**
+   * Create system intake contact in database
+   * */
+  const createContact = async (
+    contact: SystemIntakeContactProps
+  ): Promise<AugmentedSystemIntakeContact | undefined> => {
     const { euaUserId, component, role } = contact;
-    return createSystemIntakeContact({
+
+    /** New contact response from mutation */
+    const createContactResponse = await createSystemIntakeContact({
```

Comment:
The app is crashing for me here when trying to create a new contact when verifying a user.  There needs to be error handling/catch block here.  Also not sure what's happening with the payload, BE is sending a GQL 422 error for `Unprocessable Entity`.  Maybe we can sync and I can demo